app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+from __future__ import annotations
+from app.players.elves.elf import Elf
+from app.players.dwarves.dwarf import Dwarf
+from app.players.player import Player
+
+
+def calculate_team_total_rating(players_list: list[Player]) -> int:
+    return sum([player.get_rating() for player in players_list])
```

Comment:
not need to create list 
```suggestion
    return sum(player.get_rating() for player in players_list)
```